### PR TITLE
Bug 1966697: Periodic GC log to debug level

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

**Review by gamli75 — APPROVED** [review]

**Comment by YuviGold** [comment]
This log should be written only in case `DeregisterInactiveCluster` or `PermanentClustersDeletion` found any clusters.
Otherwise, it's just spammy.

/hold

File: internal/cluster/cluster.go [hunk 1]
```diff
@@ -982,9 +982,11 @@ func (m Manager) PermanentClustersDeletion(ctx context.Context, olderThan strfmt
 		return reply.Error
 	}
 	for i := range clusters {
+
 		c := clusters[i]
-		m.log.Debugf("Deleting all S3 files for cluster: %s", c.ID.String())
+		m.log.Infof("Permanently deleting cluster %s that was de-registered before %s", c.ID.String(), olderThan)
 
+		m.log.Debugf("Deleting all S3 files for cluster: %s", c.ID.String())
```
Comment (YuviGold) [inline internal/cluster/cluster.go:989]: why do we need the debug log when we have the above info log?

Comment (oshercc) [reply internal/cluster/cluster.go]: I guess it's redundant to be specific about what is being deleted,

**Review by YuviGold — APPROVED** [review]

**Pull request merged**